### PR TITLE
velvet

### PR DESCRIPTION
**Pull request by tiagoantao**
This is velvet. Interesting on its own, but also required by stacks

### CODE REVIEW

File: recipes/velvet/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  version: "1.2.10"
+build:
+  number: 0
+source:
+  fn: velvet_1.2.10.tgz
+  url: https://www.ebi.ac.uk/~zerbino/velvet/velvet_1.2.10.tgz
+requirements:
+  build:
+    - gcc
+  run:
```
Comment (bgruening) [inline recipes/velvet/meta.yaml]: I guess we agreed to include libgcc in runtime deps if they are in build deps.

Comment (tiagoantao) [inline recipes/velvet/meta.yaml]: done ;)

**Comment by bgruening** [comment]
Looks great to me, do you want to try with the os-x builds? Be warned it can open a can of worms.

**Comment by bgruening** [comment]
:+1:

**Pull request merged**